src/components/SubTab.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './SubTab.css';
+
+class SubTab extends Component {
+  render() {
+    return (
+      <div className="list-group-item " data-behaviour={ 'subtab-item-' + this.props.index } data-id={ this.props.index }>
```

Comment:
It's a good practice to destructure props at the top.
Read about Array/Object restructuring in ES6